epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/launch_plugin.py

```diff
@@ -47,14 +51,16 @@ def compile(self, *args, **opts):
         else:
             return misc.compile(nb_core=self.options['nb_core'], *args, **opts)
 
-# Phase-Space Optimization ------------------------------------------------------------------------------------
+# CUDACPP runcard block ------------------------------------------------------------------------------------
 template_on = \
 """#***********************************************************************
 # SIMD/GPU configuration for the CUDACPP plugin
 #************************************************************************
  %(cudacpp_backend)s = cudacpp_backend ! CUDACPP backend: fortran, cuda, hip, cpp, cppnone, cppsse4, cppavx2, cpp512y, cpp512z, cppauto
+#*** WARNING! Do not change the cudacpp runcard below! Users should normally change only the cudacpp_backend card ***
```

Comment:
That warning is confusing... why having those, and why include those if you do not want user to change those?
If this is not for the user, then hide those parameter (by not writting them in the run_card such that they can be added by expert user and see by advance user via "update to_all"). Note that we can have a second template with his own block name if needed